runner/Dockerfile

```diff
@@ -38,7 +38,6 @@ COPY ./requirements.txt /app
 RUN pip install --no-cache-dir -r requirements.txt
 
 RUN pip install https://github.com/chengzeyi/stable-fast/releases/download/v1.0.3/stable_fast-1.0.3+torch211cu121-cp311-cp311-manylinux2014_x86_64.whl
-
```

Comment:
nit: no need to commit this whitespace change

Reply:
Done